src/cfnlint/data/AdditionalSpecs/Exclusive.json

```diff
@@ -53,12 +53,6 @@
       ]
     },
     "AWS::RDS::DBInstance": {
-      "DBSnapshotIdentifier": [
```

Comment:
This needs to be converted to another check as DBSnapshotIdentifier can be an empty string which means CloudFormation will use the Username and Password fields.